src/core/common/errors.ts

```diff
@@ -136,6 +136,12 @@ export enum ERROR_CODES {
    */
   PASSWORD_TOO_SHORT = "PASSWORD_TOO_SHORT",
 
+  /**
+   * PASSWORD_INCORRECT is returned when a logged in operation that requires the
+   * password returns the wrong password.
+   */
+  PASSWORD_INCORRECT = "PASSWORD_INCORRECT",
```

Comment:
I've found it is usually more secure if you're vague about what was incorrect when handling a login request.

Does specifying that the password specifically was incorrect make sense?

Reply:
This particular error is only thrown when a user is already logged in, and is being asked to provide their password for confirmation as is explicitly referenced in the enum description. This is why we have a different (more vague) error code for standard auth errors `INVALID_CREDENTIALS`.